src/MyHealth/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent, useState } from "react"
+import { View, StyleSheet, ScrollView } from "react-native"
+import SegmentedControl from "@react-native-community/segmented-control"
+import { useTranslation } from "react-i18next"
+
+import { useStatusBarEffect } from "../navigation"
+import { GlobalText, StatusBar } from "../components"
+import Today from "./Today"
+import OverTime from "./OverTime"
+
+import { Typography, Colors, Spacing } from "../styles"
+
+const MyHealthScreen: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.primaryLightBackground)
+  const { t } = useTranslation()
+  const [selectedIndex, setSelectedIndex] = useState<number>(0)
+
+  const determineDisplayToday = () => {
```

Comment:
Minor thought building off of Amanda's comment of using a specific type for the tabs.

```suggestion
  const determineTabStyle = (myHealthTab: MyHealthTab) => {
     if (activeTab === myHealthTab) {
        return {}
     } else {
       return style.hidden
     }
     
     ...
             <View style={determineTabStyle(TodayTab)}>
          <Today />
        </View>
        <View style={determineTabStyle(OverTimeTab)}>
          <OverTime />
        </View>
```